std_msgs/Float32MultiArray stateVec
std_msgs/Float64MultiArray viz
---
int64 direction
